config.lua

```diff
@@ -26,17 +26,19 @@ Config.RemoveLockpickAdvanced = 0.2 -- Chance to remove advanced lockpick on fai
 Config.CarJackEnable = true -- True allows for the ability to car jack peds.
 Config.CarjackingTime = 7500 -- How long it takes to carjack
 Config.DelayBetweenCarjackings = 10000 -- Time before you can carjack again
+Config.PedDefenseChance = 0.6 -- Probability (0-1) that a ped will defend themselves
+Config.PedAccuracy = 50 -- How accurate the ped's shooting will be (1-100)
 Config.CarjackChance = {
     ['2685387236'] = 0.0, -- melee
-    ['416676503'] = 0.5, -- handguns
-    ['-957766203'] = 0.75, -- SMG
-    ['860033945'] = 0.90, -- shotgun
-    ['970310034'] = 0.90, -- assault
-    ['1159398588'] = 0.99, -- LMG
-    ['3082541095'] = 0.99, -- sniper
-    ['2725924767'] = 0.99, -- heavy
+    ['416676503'] = 100, -- handguns
```

Comment:
Is this intended? `math.random()` only generates a float from 0 -> 1

Reply:
It was unintended for the actual carjackchance variables and was an oversight could be set to Default values.